test.rb

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
+require 'logatron/logatron'
+
+Logatron.info("Log entry from the 'testing script.'  Single quotes in the body of this message should be converted into double quotes.")
```

Comment:
Everyone who comes across this file is going to ask the same question Erik just did. Is there not another way you can test your logging output from a docker build? It is unusual to have a file like this in a repo.